src/store/store.js

```diff
@@ -1,17 +1,17 @@
-import Vuex from 'vuex';
-import Vue from 'vue';
+import Vuex from 'vuex'
+import Vue from 'vue'
 
 Vue.use(Vuex)
 
 export default new Vuex.Store({
   state: {
-    user: null,
+    user: 'Ryohei',
```

Comment:
Remove hardcoded values in your commits. If you want to manipulate Vue/Vuex data, use your dev tools.